workflows/prognostic_c48_run/emulation/emulation/_filesystem.py

```diff
@@ -0,0 +1,43 @@
+# keras has several routines which interact with file paths directly as opposed to
+# filesystem objects, which means we need these wrappers so we can allow remote paths
+
+import contextlib
+import tempfile
+import fsspec
+import os
+
+
+@contextlib.contextmanager
```

Comment:
This file was copied out of fv3fit because I wasn't sure if we would have the dependency installed.  Could delete and link to fv3fit, or just pare down to the `get_dir`, which is the only thing used in `emulate.py`

Reply:
> just pare down to the get_dir, which is the only thing used in emulate.py

This sounds like a good idea.